core-tests/src/test/java/org/mule/MuleEventTestCase.java

```diff
@@ -147,17 +138,17 @@ public void testEventSerializationRestart() throws Exception
 
     }
 
+    @Ignore("Credentials were provided by the inbound endpoint")
```

Comment:
seems to be than rather ignoring, this test should go away. All credentials will now be handled at the ConnectionProvider level right?